charts/ckan/values.schema.json

```diff
@@ -0,0 +1,259 @@
+              },
+              "size": {
+                "$ref": "#/$defs/quantity"
+              },
+              "storagePath":{
+                "type": "string"
+              }
+            }
+          },
+          "sysadmin": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "password": {
+                "type": "string"
+              },
+              "email": {
+                "$ref": "#/$defs/email"
+              }
+            }
+          },
+          "smtp": {
+            "type": "object",
+            "properties": {
+              "server": {
+                "type": "string"
+              },
+              "user": {
+                "type": "string"
+              },
+              "password": {
+                "type": "string"
+              },
+              "mailFrom": {
+                "$ref": "#/$defs/email"
+              },
+              "tls": {
+                "type": "string",
+                "enum": [
+                  "enabled",
+                  "disabled"
+                ]
+              },
+              "starttls": {
+                "type": "boolean"
+              }
+            }
+          },
+          "podSecurityContext": {
+            "type": "object"
+          },
+          "securityContext": {
+            "type": "object"
+          },
+          "podAnnotations": {
+            "type": "object"
+          },
+          "resources": {
+            "type": "object"
+          }
+        }
+      },
+      "datapuscher": {
+        "type": "object",
+        "properties": {
+          "replicas": {
+            "type": "integer"
+          },
+          "image": {
+            "$ref": "#/$defs/image"
+          },
+          "podSecurityContext": {
+            "type": "object"
+          },
+          "securityContext": {
+            "type": "object"
+          },
+          "podAnnotations": {
+            "type": "object"
+          },
+          "resources": {
+            "type": "object"
+          }
+        }
+      },
+      "postgresql": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "ckanDbs": {
+          "type": "object",
+          "properties": {
+            "datastoreUsername": {
+              "type": "string"
+            },
+            "datastorePassword": {
+              "type": "string"
+            },
+            "datastoreDb": {
+              "type": "string"
+            },
+            "datapusherUsername": {
+              "type": "string"
+            },
+            "datapusherPassword": {
+              "type": "string"
+            },
+            "datapusherDb": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "redis": {
+        "enabled": {
+          "type": "boolean"
+        }
+      },
+      "solr": {
+        "enabled": {
+          "type": "boolean"
+        }
+      }
```

Comment:
That's not how that works, missing the `properties` field and `additionalProperties: false` as well